app/services/market_analysis_core.py

```
@@ -682,7 +701,14 @@ async def complete_market_assessment(
                 "arbitrage_opportunities": results[3] if len(results) > 3 and not isinstance(results[3], Exception) else None,
                 "alpha_signals": results[4] if len(results) > 4 and not isinstance(results[4], Exception) else None
             }
-            
+
+            try:
+                yield_data = await market_data_feeds.get_yield_opportunities(symbol_list)
+                assessment["yield_opportunities"] = yield_data
+            except Exception as yield_error:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)